genestack_client/files_util.py

```diff
@@ -194,6 +194,32 @@ def unlink_file(self, accession, parent):
         """
         self.invoke('unlinkFile', accession, parent)
 
+    def link_files(self, accessions, parents):
```

Comment:
I agree that the structure isn't very nice. How about this: a `Map<String,List<String>>` like 
```
{
   parent1: [child1, child2]
   parent2: [child1]
   ...
}
```